_posts/2021-01-13-hapi-fhir.md

```diff
@@ -2,8 +2,8 @@
 layout: post
 authors: [youri_vermeir]
 title: 'Introduction to HAPI FHIR'
-image: /img/2021-01-23-hapi-fhir/hapi-logo.jpg
-tags: [eHealth, FHIR]
+image: /img/2021-01-23-hapi-fhir/hapi.png
```

Comment:
If you go the main page of the blog with the overview of all blog posts, then this image looks quite big compared to the others. It also stretches the layout of the overview page. Can you have a look what goes wrong?